app/services/organizations/importers/google_csv_import_service.rb

```diff
@@ -13,6 +13,7 @@ def initialize(file)
       end
 
       def call
+        # Validate that file is in CSV format before we read from it.
```

Comment:
@khiga8 Thanks for pointing this out! This started a discussion in our last meeting about other validations we should be checking for in this service. I opened PR #1211 which adds a file type validation and checks for the correct headers.

Hopefully I didn't step on your toes, but I figured the validations were outside the scope of this issues and didn't want to cause any confusion or add more work. If you have a better way to validate though I'd love to see it!!

The controller

Reply:
This is great! Thank you :)